docs/app/pages/components/sections/tabs/card-tabs-ng1/card-tabs-ng1.component.ts

```diff
@@ -0,0 +1,27 @@
+import { Component } from '@angular/core';
+import { DocumentationSectionComponent } from '../../../../../decorators/documentation-section-component';
+import { ICodePenProvider } from '../../../../../interfaces/ICodePenProvider';
+import { ICodePen } from '../../../../../interfaces/ICodePen';
+
+@Component({
+    selector: 'uxd-components-card-tabs-ng1',
```

Comment:
Most sections should be able to specify a change detection strategy of OnPush to give us a bit better page performance